pkg/rbac/messagesDev.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Contributors to the Open Cluster Management project
```

Comment:
Can we use `_test.go` in the file name?

Reply:
I am getting `pkg/resolver/messages_test.go:20:25: undefined: rbac.NewMockCacheForMessages` error if I do that.